Automattic/jetpack
# Contact Form: Adjust Form placeholder colors

Fixes #27985

#### Changes proposed in this Pull Request:
* Adjusts the Form placeholder icons and link colors for better consistency

Before
![image](https://user-images.githubusercontent.com/7811225/208989212-9e15924a-ed5a-4e73-8371-355b22a7350c.png)

After
![image](https://user-images.githubusercontent.com/7811225/208988905-53aa1fb8-7f29-4d29-8ef5-cd98a77a65e9.png)


#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Create a post and add a Form block
* Check if the icons and link colors are more consistent now
